lib/NUPACK/emscripten/FullEnsemble.cpp

```diff
@@ -180,20 +178,23 @@ FullAdvancedResult* FullEnsembleNoBindingSite(const std::string& seqString, int
     }
 
     //initialize the result to return
+    
     FullAdvancedResult* result = new FullAdvancedResult();
     
     std::string mfeStructure;
     //get dot bracket notation from data 
     for ( int i = 0; i < suboptStructs.nStructs; i++ ) {
         oneDnaStruct currentStruct = suboptStructs.validStructs[i];
-
+        
         //get the secondary strucutre in dot paren notation 
-        std::string singlestructure = getDotParens(pseudoknotted, suboptStructs.seqlength, &currentStruct);
+        //old method
+        //std::string singlestructure = getDotParens(pseudoknotted, suboptStructs.seqlength, &currentStruct);
         
+        std::string singlestructure = getDotParens(pseudoknotted, suboptStructs.seqlength, &currentStruct);
```

Comment:
Nothing changed here?

Reply:
this is left over from me tryin gout the other dot parens process to see if it gave a time boost that was worth putting time into understanding for implementation. Those comments are gone now.